### ISSUE TITLE
"name" can't be empty FIX

### ISSUE THREAD

**Issue by benjamano** [MattIPv4/PyDMXControl#59]
Hi All,

Recently, I've run into an error using the OpenDMXController.
When running it gives me an error along the lines of " 'name' can be empty".

**This error originated from a file inside the PyDMXControl Library, so here is the solution:**
```
C:\Users\<Your Name>\AppData\Roaming\Python\Python312\site-packages\PyDMXControl\web\_routes.py
```
OR
```
C:\Users\<Your Name>\AppData\Local\Programs\Python\Python312\Lib\site-packages\PyDMXControl\web\_routes.py
```

Depending on where Python is installed, choose whichever one above is correct.

Edit the line:
```
 routes = Blueprint('', __name__, url_prefix='/')
```
To:
```
 routes = Blueprint('main', __name__, url_prefix='/')
```
And this should solve the bug!
Hopefully this helps anyone else who's run into this problem.

**Comment by MattIPv4** [comment]
Dupe of #45, this was fixed in #53 but I don't think I've had the time to release a new version since. I'll add it to my list of things to do 👍

**Comment by benjamano** [comment]
I can make a pull request later today if you'd like (:
